### DISCUSSION TITLE
using Customize ChatGPT Prompt have issue

### DISCUSSION THREAD

**Discussion by eu09** [danny-avila/LibreChat#112] [Troubleshooting] [answered]
I got this error when I using Customize ChatGPT Prompt

An error occurred. Please try again in a few moments.

Error message: Cannot read properties of undefined (reading 'includes')

**Comment by danny-avila** [comment]
Thanks I will take a look soon. Are you on main?

**Comment by danny-avila** [comment, marked as answer]
### @eu09 I can confirm this is not an issue on the main branch. make sure you're on it

![image](https://user-images.githubusercontent.com/110412045/226906905-0768221f-fbb6-402c-8a05-5e518618cb09.png)
